server/testdata/certs/localhost.crt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
These certs expire in 2021. Could you regenerate them with the `--expires` flag set to like 100 years?

Reply:
good call, done